web/spec/gotrue.json

```diff
@@ -11058,8 +11059,17 @@
 				}
 			],
 			"type": {
-				"type": "literal",
-				"value": "sms"
```

Comment:
There are some gotrue type changes here. Is this intentional? Split it up into a different PR?

Reply:
this us auto gened, so I think someone forgot to run `npm run gen:supabase` in a previous PR?